storm/src/main/storm/mesos/NimbusScheduler.java

```diff
@@ -102,7 +102,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
       case TASK_LOST:
       case TASK_ERROR:
         LOG.info("Received status update: {}", taskStatusToString(status));
```

Comment:
@JessicaLHartog 

> `TASK_LOST`, `TASK_FAILED`, and `TASK_ERROR` can use a warn log line

`TASK_LOST` *used* to be a normal terminal state for our tasks.  It doesn't *seem* to be anymore, and I'm not sure about the reason.  So the last 2 definitely deserve WARN level logs, but I'm not sure about `TASK_LOST`, need to think more.